### ISSUE TITLE
build 5.7.2 & BIMI check

### ISSUE THREAD

**Issue by Pascal76** [domainaware/checkdmarc#148]
checkdmarc email.voyageprive.es

returns : 
"bimi": {
    "record": null,
    "valid": false,
    "selector": "default",
    "error": "A BIMI record does not exist at the default selector for this domain **or its base domain**"
  }

while there is a BIMI record on (_bimi.)voyageprive.es

**Comment by EmailKarma** [comment]
Maybe DNS lookup failure... I see the output when I test - looks like your cert expired on 10/02/2024

```
 "bimi": {
    "record": "v=BIMI1; l=https://images.bovpg.net/crm/VP_GLYPH_170x170.svg; a=https://images.bovpg.net/crm/bimi/voyage_priv__.pem;",
    "valid": true,
    "selector": "default",
    "tags": {
      "v": {
        "value": "BIMI1"
      },
      "l": {
        "value": "https://images.bovpg.net/crm/VP_GLYPH_170x170.svg"
      },
      "a": {
        "value": "https://images.bovpg.net/crm/bimi/voyage_priv__.pem"
      }
    },
    "image": {
      "svg_version": "1.2",
      "base_profile": "tiny-ps",
      "title": "bimi-svg-tiny-12-ps",
      "width": 145.0,
      "height": 145.0,
      "filesize": "0.818 KB",
      "sha256": "4b21890014bad1b79d908ca7b50d5508b9f439e7de50e29c075367413bf7c561"
    },
    "certificate": {
      "issuer": {
        "C": "US",
        "O": "DigiCert, Inc.",
        "CN": "DigiCert Verified Mark RSA4096 SHA256 2021 CA1"
      },
      "subject": {
        "jurisdictionC": "FR",
        "jurisdictionST": "Provence-Alpes-Côte d'Azur",
        "jurisdictionL": "Aix-en-Provence",
        "businessCategory": "Private Organization",
        "serialNumber": "479 345 043",
        "C": "FR",
        "L": "Aix-en-Provence",
        "street": "330 Rue Pascal Duverger",
        "O": "VPG SAS",
        "CN": "VPG SAS",
        "UNDEF": "016445819"
      },
      "serial_number": "2466617948829818592410305399428204101",
      "expires": "2024-10-02 23:59:59Z",
      "valid": false,
      "domains": [
        "voyage-prive.com"
      ],
      "logotype_sha256": "4b21890014bad1b79d908ca7b50d5508b9f439e7de50e29c075367413bf7c561",
      "validation_errors": [
        "certificate has expired",
        "voyageprive.es does not match the certificate domains, ['voyage-prive.com']"
      ]
    },
    "warnings": [
      "Most providers will not display a BIMI image without a valid mark certificate"
    ]
  }
```

**Comment by Pascal76** [comment]
did you test checkdmarc email.voyageprive.es or checkdmarc voyageprive.es ?

**Comment by EmailKarma** [comment]
I can't recall now, but I'm seeing different results (failed) for the email subdomain than I am (pass) for the org domain now.

**Comment by seanthegeek** [comment]
Fixed in 5.7.5

@Pascal76 Note: the mark verification certificate referenced in the BIMI record for `voyageprive.es` has expired, and is only valid for `voyage-prive.com`, not `mail.voyageprive.es` or `voyageprive.es`. Domains and subdomains in certificates must match exactly. certificate authorities do offer to have multiple domains in the same certificate for an additional fee.

**Comment by Pascal76** [comment]
Thank you @seanthegeek 
I know for the mismatch ... but it is an opportunity for you to do tests on an invalid case :)
